<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUITimeHistoryRBODYDialog</class>
 <widget class="QWidget" name="GUITimeHistoryRBODYDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButton_Cancle">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_OK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>153</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_Name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Upper">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_Type"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Time History RBODY Variable</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_FX">
            <property name="text">
             <string>FX</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBox_FY">
            <property name="text">
             <string>FY</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBox_FZ">
            <property name="text">
             <string>FZ</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_FXI">
            <property name="text">
             <string>FXI</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_FYI">
            <property name="text">
             <string>FYI</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="checkBox_FZI">
            <property name="text">
             <string>FZI</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_MX">
            <property name="text">
             <string>MX</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBox_MY">
            <property name="text">
             <string>MY</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkBox_MZ">
            <property name="text">
             <string>MZ</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_MXI">
            <property name="text">
             <string>MXI</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBox_MYI">
            <property name="text">
             <string>MYI</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="checkBox_MZI">
            <property name="text">
             <string>MZI</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBox_RX">
            <property name="text">
             <string>RX</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBox_RY">
            <property name="text">
             <string>RY</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCheckBox" name="checkBox_RZ">
            <property name="text">
             <string>RZ</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
